politeiawww/cmd/politeiawwwcli/commands/newinvoice.go

```diff
@@ -221,8 +221,8 @@ func validateParseCSV(data []byte) (*v1.InvoiceInput, error) {
 
 	// Validate that the invoice is CSV-formatted.
 	csvReader := csv.NewReader(strings.NewReader(string(data)))
-	csvReader.Comma = www.PolicyInvoiceFieldDelimiterChar
-	csvReader.Comment = www.PolicyInvoiceCommentChar
+	csvReader.Comma = v1.PolicyInvoiceFieldDelimiterChar
```

Comment:
Can we use `cms` instead of `v1` as the package identifier?  `v1` is ambiguous since both pi and cms are `v1`.

Reply:
Right, nice catch mate. Will commit the changes! 👍